certification/internal/policy/operator/validate_operator_bundle.go

```diff
@@ -35,11 +37,27 @@ func (p ValidateOperatorBundleCheck) Validate(ctx context.Context, bundleRef cer
 	return p.validate(ctx, report)
 }
 
-func (p ValidateOperatorBundleCheck) getDataToValidate(ctx context.Context, imagePath string) (*operatorsdk.OperatorSdkBundleValidateReport, error) {
+func (p *ValidateOperatorBundleCheck) dataToValidate(ctx context.Context, imagePath string) (map[string]bool, error) {
+	valid := make(map[string]bool)
+	bundle, err := manifests.GetBundleFromDir(imagePath)
+	if err != nil {
+		return nil, fmt.Errorf("could not read bundle: %v", err)
+	}
+	validators := validation.DefaultBundleValidators
+	validators = validators.WithValidators(validation.ObjectValidator)
+	results := validators.Validate(bundle.ObjectsToValidate()...)
```

Comment:
Nice!

Reply:
Ha... That wasn't supposed to make it into this patch. lol

Reply:
Whoops!